src/impl.cpp

```diff
@@ -561,7 +568,7 @@ void Manifold::Impl::CalculateNormals() {
       // should just exclude it from the normal calculation...
       if (!la::isfinite(currEdge[0]) || !la::isfinite(prevEdge[0])) return;
       double dot = -la::dot(prevEdge, currEdge);
-      double phi = dot >= 1 ? 0 : (dot <= -1 ? kPi : std::acos(dot));
+      double phi = dot >= 1 ? 0 : (dot <= -1 ? kPi : acos(dot));
```

Comment:
maybe we should call these floats, since our acos function isn't giving us double precision?

Reply:
I guess what I mean is, our normals aren't really as precise as doubles anymore right? It's probably fine to leave it as is, but I was just thinking maybe it would be good to remind ourselves that these have more error than e.g. our positions.